### DISCUSSION TITLE
Connect to SQL Server named instance

### DISCUSSION THREAD

**Discussion by DrorSegev** [mindsdb/mindsdb#1264] [Get help]
My Sql server is installed as a named instance:
RAM-SQLDEV-W16\RAMSQLQA16
How do I connect to this DB?
What should I put in Host?

Thanks 
Dror

**Comment by ZoranPandovski** [comment]
Hi @DrorSegev I think you should specify host then named instance as APPHOST\RAM-SQLDEV-W16\RAMSQLQA16

**Comment by DrorSegev** [comment]
@ZoranPandovski 
Hi,
RAM-SQLDEV-W16 - is the host:
See connection string setting in SSMS:
![image](https://user-images.githubusercontent.com/12187079/119250689-7f545600-bbaa-11eb-9070-fb8998f69870.png)

Connection string in mindsdb (not working):
![image](https://user-images.githubusercontent.com/12187079/119250716-a743b980-bbaa-11eb-900d-ede72854db57.png)


Thanks
Dror

**Comment by ZoranPandovski** [comment]
@DrorSegev that looks ok, it should work. Few things:
- Check if the port is correct
- Check if the database name is correct
If those are correct can you share the log error from the MindsDB Console when you click on a connect?

**Reply by ZoranPandovski** [reply]
Hello @DrorSegev did you manage to connect?

**Comment by DrorSegev** [comment]
hi. no...
for meatime I've created a instance with no name.
Thanks

**Reply by ZoranPandovski** [reply]
Ok, great. We will check what is the exact way for connecting to the `named` instaces.
